e2e_samples/config_driven_data_pipelines/.coveragerc

```diff
@@ -0,0 +1,13 @@
+[run]
```

Comment:
Hi @thurstonchen , do you think we still need this?

Reply:
Hi @gary918 ,
Ah~~ thanks for your reminder!
Just removed _.coveragerc_ file :-)
Best,
Thurston